Translation/en/RedirectedResources/assets/abdata/adv/scenario/c06/31/2/zz_machineTranslation.txt

```diff
@@ -1 +1,48 @@
+「うん、きみのことが好きなんだ。俺と付き合ってほしい」=Yes, I like you. I want you to go out with me.
+「それは……そうですか。私のことを、好いてくださっていたのですね」=That's ...... true. You like me, don't you?
+「なんとなく伝わってたかな？」=Did you get the gist of it?
+「はい。私を女性として意識してくださっていることは、わかっておりました」=Yes. I've always known that you were interested in me as a woman.
+「ですが、よろしいのですか？　知っての通り、私は古風な女です。貴方とお付き合いするには、重すぎるかもしれません」=But are you sure?　As you know, I'm an old-fashioned woman. I may be too heavy for you.
+「重い？」=Heavy?
+「いちいち、男女関係の考え方が古いのです。今の若い人たちは結婚を前提にしてお付き合いなどしないでしょう」=One, two, three, four, five. Young people today don't go out with the intention of getting married.
+「ですが私は、家庭を築け、子を成せと言われて育った女です。そんな私が貴方の重荷になってしまわないかが心配なのです」=But I am a woman who grew up being told to start a family and have children. But I am a woman who grew up being told to raise a family and have children, and I am worried that I will become a burden to you.
+「今までの話を聞いていればなんとなく覚悟はできてるよ。いや……なんとなくなんて心持ちじゃ、甘いのかな」=I'm not going to destroy [H名]. I'm not sure if ...... is too much to ask.
+「いいえ、貴方なりに考えていただけたなら、それで十分です。私と人生を共にしていただけるのですね？」=No, if you think about it, that's good enough for me. You will share your life with me, won't you?
+「是非。あ、でも、家族の人たちは嫌がるかな。俺みたいな一般人じゃ釣り合わないって言われたりしない？」=By all means. I don't know if my family will like it, though. Wouldn't they say I'm not good enough for them?
+「その点はご心配なく。私の連れて来た殿方に文句があるというのでしたら、私は祖母であろうと戦います」=─ Don't worry about that. If you have a problem with the gentleman I brought, I will fight him, even if he is my grandmother.
+「た、戦う！？」=Fight!
+「貴方の魅力は私がいちばんよく理解しております。それをわからないというのであれば私が家族に教えて差し上げればよいだけの話です」=I understand your charm better than most. If you don't understand that, then all you have to do is tell your family about it.
+「そのうえで首を縦に振らないのであれば、私は家族と矛を交える覚悟をしております」=If you don't shake your head after that, I'm prepared to contradict my family.
+「いや、暴力はちょっと……」=No, I'm not into violence. ......
+「ふふふ、ご安心ください。暴力よりももっと切れ味のよい、言葉の矛でございますから」=Mmm-hmm, don't worry. Don't worry, I'm a verbal contraption that is much sharper than violence.
+「第一、私は家族から、どこに出しても恥ずかしくない娘として育ったと太鼓判を押してもらっております」=First of all, my family is convinced that I've grown up to be a daughter who can be shown anywhere.
+「そんな女が連れて来た殿方を認められないのは、筋が通っておりません。そこをしっかりと問いただし、貴方をお守りいたしましょう」=It doesn't make sense that you can't approve of a man brought by such a woman. I will protect you by questioning her about it.
+「心強いような、不穏なような……」=That's both reassuring and disturbing. ......
+「うふふふ、大丈夫です。万が一、文句を言われたらの話です。私の家族はそこまで強情な方々ではございません」=It's okay. It's just in case they complain. My family is not that stubborn.
+「それならいい、のかな。あ、もし結婚するってなったら婿養子になるのかな？」=I'm not sure if that's a good idea. Oh, if I get married, will I be adopted by my son-in-law?
 「とんでもございません、女はいずれ嫁ぐものです。結婚をするとき私は貴方の姓になります、それは私の長年の夢でもございます」=No way, a woman has to marry sooner or later. When I get married, I will take your surname, which has been my dream for many years.
+「[H名前]が[P姓] [H名]になるのか」=[H名] will become [P last name] [H first name]?
```

Comment:
Uh, oh. These will need to be corrected. Looks like you caught the first one.

Reply:
Fixed the tags